posts/TechYesWeCan_Pt1.md

```diff
@@ -0,0 +1,115 @@
+
+<responsive-figure src="https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/10/Leigha_IMG1.png">
+
+*The world needs more women leaders - why not you?*
+
+**The Shape of you - Building Your Personal Brand**
+
+Being ahead in the world no longer starts with “who do you know” or “what do you know” but “who knows you”. Networking and creating your own personal brand is a top priority. Making the best first impression possible, then leveraging that to build personal opportunities. Be true to yourself so you don’t come off as disingenuous. 
+
+Having a personal brand doesn’t mean you need to have a second personality. Are you the same online as you are in person? Should you be?
+
+It’s ok to start over. Think of “re-branding” yourself as a seasonal change, a metaphorical closet if you will. Take all your clothes out of the drawers and off the hangers and put them all on the floor. Wow -- big pile. Decide what you want to keep and what you don’t *need* to keep. 
+
+*Get comfortable with being uncomfortable.*
+
+**Nancy Douyon | International User Experience Research Lead | Uber**
+
+I love when a speaker starts their talk with a simple image and questions to the audience! It usually means there’s a twist and a fun talk about to begin -- and that’s just what we got from Nancy Douyon. What would you change about this haitian house? Answers rang out about electricity, paint jobs, siding, and the roof. For each answer Nancy gave a valid and thought-provoking reason why that wasn’t necessary or good for the house in that part of the world. And this was just the beginning of a talk that made me realize everything I don’t think about when I start to brainstorm solutions to a problem.
+
+*Good solutions require insight and data into that problem.*
+
+Nancy then went on to describe how the world is changing and how building and designing for that world requires a different way of things. We as engineers need to be building for a more inclusive world. One fabulous example she gave was changing stations only existing in womens restrooms. Who thought that women would be the only ones out and about needing to change a baby's diaper? But then again -- why would it cross your mind as any other way? It’s the way it’s always been. We need to change our thoughts, test and design differently.
+ 
+
+As I’ve said in past articles, this conference only gets better year by year. They have an amazing group of women (and men) that take each attendees thoughts into account and put that into the following year’s conference. The speakers always blow me away, but I was struck extra hard this year; these keynotes really hit it home. They all had something totally different to say but with one major thought -- build yourself up! Be the best you that you can be by getting to know yourself better.
+
+
```

Comment:
Use a proper separation (multiple empty lines all end up being just one)

Reply:
Got it